programs/virtual-curve/src/state/virtual_pool.rs

```diff
@@ -53,6 +53,24 @@ pub enum PoolType {
     Token2022,
 }
 
+#[repr(u8)]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    PartialEq,
+    IntoPrimitive,
+    TryFromPrimitive,
+    AnchorDeserialize,
+    AnchorSerialize,
+)]
+pub enum MigrationProgress {
```

Comment:
Can you add comments on the `MigrationProgress`? Maybe something like
```
// Pool state transition flows:
// 1. Without jup lock
//    PreBonding -> LockedVesting -> CreatedPool
//
// 2. With jup lock
//    PreBonding -> PostBonding -> LockedVesting -> CreatedPool
```

Reply:
Fixed